ariadne/src/Ariadne/Wallet/Knit.hs

```diff
@@ -4,28 +4,47 @@ import Universum
 
 import qualified Data.ByteArray as ByteArray
 
+import Control.Lens (makePrisms)
 import IiExtras
+import Pos.Client.Txp.Util (defaultInputSelectionPolicy)
 import Pos.Crypto.Hashing (hashRaw, unsafeCheatingHashCoerce)
 import Pos.Crypto.Signing (emptyPassphrase)
 import Serokell.Data.Memory.Units (fromBytes)
 import Text.Earley
 
 import Ariadne.Cardano.Knit (Cardano, ComponentValue(..), tyTxOut)
+import Ariadne.Cardano.Orphans ()
 import Ariadne.Wallet.Face
 
 import Knit
 
 -- Component type for Knit
 data Wallet
 
-data instance ComponentValue _ Wallet
+data instance
+     ComponentValue _
+       Wallet = ValueInputSelectionPolicy !InputSelectionPolicy
```

Comment:
This formatting is a little bit weird if you ask me.

Reply:
`hindent` :man_shrugging: